assets/model_monitoring/components/src/model_monitor_compute_histogram/histogram.py

```diff
@@ -130,19 +130,17 @@ def _to_bin_edges(histogram_buckets: pyspark_sql.DataFrame):
 
 
 def compute_histograms(
-    df: pyspark_sql.DataFrame, histogram_buckets: pyspark_sql.DataFrame
-) -> tuple:
+        df: pyspark_sql.DataFrame,
+        histogram_buckets: pyspark_sql.DataFrame,
+        override_numerical_features: str,
+        override_categorical_features: str)-> tuple:
     """Compute data drift measures and perform tests."""
     # Generate histograms only for columns in both baseline and target dataset
-    columns_dict = {}
-    df_dtypes = dict(df.dtypes)
-    for (column_name, data_type) in df_dtypes.items():
-        columns_dict[column_name] = data_type
-
-    numerical_columns = get_numerical_cols_with_df(columns_dict,
-                                                   df)
-    categorical_columns = get_categorical_cols_with_df(columns_dict,
-                                                       df)
+    numerical_columns_names, categorical_columns_names = get_numerical_and_categorical_cols(
+                                                            baseline_df,
```

Comment:
where comes this baseline_df? And the below common_columns_dict?

Reply:
sorry I did not update this file. It is the old version.